.bazelrc

```diff
@@ -0,0 +1,410 @@
+# The following .bazelrc content is forked from the main Envoy repository.                  # unique
```

Comment:
Looks like this file was actually cloned from Nighthawk, not from Envoy. That is fine to do, since we need to be able to build Nighthawk, but we should update the commentary in this file.

Reply:
@mum4k Sure, I'll chang Envoy to Nighthawk.